docs/support/tutorials/index.md

```diff
@@ -57,10 +57,12 @@
 * [Starting with parallel R](parallel-r.md)
 
 ## Geoinformatics
+* [Geocomputing in CSC computing environment](https://research.csc.fi/geocomputing)
```

Comment:
Was the intention to link this one maybe: https://csc-training.github.io/geocomputing_course/ ? The currently linked page is no tutorial :thinking:

Reply:
That is a little bit matter of taste. For example the DA-guide is not much different. I would like to include the link here, could think of having then some better describing link text..

Reply:
Yes updating the link text might help

Reply:
Changed the link text.